gravitee-apim-plugin/gravitee-apim-plugin-entrypoint/gravitee-apim-plugin-entrypoint-sse/src/main/resources/schemas/schema-form.json

```diff
@@ -2,6 +2,13 @@
   "type": "object",
   "id": "urn:jsonschema:io:gravitee:plugin:entrypoint:sse:configuration:SseEntrypointConnectorConfiguration",
   "properties": {
+    "heartbeatIntervalInMs": {
+      "type": "integer",
+      "default": 5000,
+      "exclusiveMinimum": 0,
```

Comment:
Should we restrict the minimum to 2000 or 3000 just to avoid miss-configuration and having bad config that coult potentially be CPU starve (ex: 1ms)

Reply:
Good point, will update the settings accordingly.